browser/extensions/brave_component_loader.cc

```diff
@@ -112,7 +114,10 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
   // it would cause a bug with loading brave://wallet not loading
   // if a tab is left open and you restart the browser.  That would
   // need to be fixed first.
-  if (ExtensionPrefs::Get(profile_)->
+  // Does not load if project id is not configured.
```

Comment:
Backwards compat, so it doesn't attempt to reload if it was loaded prior to upgrade. This may be overkill for debug builds.

Reply:
I'm ok with that